content/en/docs/concepts/services-networking/service.md

```diff
@@ -934,29 +931,49 @@ Service is a top-level resource in the Kubernetes REST API. More details about t
 API object can be found at:
 [Service API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
 
-## SCTP support
+## Supported protocols {#protocol-support}
+
+{{< note >}}
+Layer 7 protocols such as HTTP are handed via the {{< glossary_tooltip term_id="ingress" >}}
```

Comment:
Is this true for services that need to communicate within the cluster? Services also can handle that right?

Reply:
The `Service` object doesn't (at least in v1.13) set up anything that understands a particular application protocol, and that's what I'm trying to say here.

If you needed to expose a webserver in a way that relies on understanding TLS, or HTTP, then `Service` is not (yet?) for you. `Ingress` is your friend.

Reply:
If you want to expose something that talks HTTP, and the means of exposure is to publish a TCP port and IPv4 address, the fact it's HTTP isn't important or relevant to `Service`.

Reply:
Maybe it's worth clarifying that you can still expose an HTTP service at the TCP layer?

Reply:
Reworded (and meaning changed)

Reply:
Thanks!